src/components/PageContent.tsx

```diff
@@ -1,18 +1,18 @@
 import { FC } from "react";
-import { PortableText, PortableTextReactComponents, PortableTextTypeComponentProps } from "@portabletext/react";
+import { PortableText } from "@portabletext/react";
```

Comment:
same deal, move `PortableText` import next to `PortableTextReactResolvers` from utils/react of the resolver package